contribution/mobile/zh-TW/procedural-names.json

```diff
@@ -257,38 +257,17 @@
 	"EQUIPMENT_TYPES_INSTRUCTIONS": "須要最少10個部件以啟用",
 	"EQUIPMENT_TYPES": {
 		"PRIMARY_WEAPON": {
-			"names": ["手槍", 
-				  "射線槍",
-				  "爆能手槍",
```

Comment:
Basically like this one (which are the line that you erased before)

Reply:
ok, I will do that next time. But how do edit the committed file?